doc/classes/Animation.xml

```diff
@@ -561,7 +561,7 @@
 			<param index="1" name="path" type="NodePath" />
 			<description>
 				Sets the path of a track. Paths must be valid scene-tree paths to a node and must be specified starting from the parent node of the node that will reproduce the animation. Tracks that control properties or bones must append their name after the path, separated by [code]":"[/code].
```

Comment:
```suggestion
				Sets the path of a track. Paths must be valid scene-tree paths to a node and must be specified starting from the [member AnimationMixer.root_node]. that will reproduce the animation. Tracks that control properties or bones must append their name after the path, separated by [code]":"[/code].
```
If the root node is mentioned, it should be here as @lyuma suggested.

Reply:
That's what the first version of my commit said!!!?!!!

Reply:
That was inappropriate because it was written in the root_motion_track location.

Reply:
you just asked me to update my PR to clarifying that the parent node was the root_node... that's what my first commit said.

This doesn't need to be this difficult....you have to understand people read these docs because they don't know....you are reviewing them as people who already know. This isn't documenting the class's source code, it's documenting how to use the class. What I wrote was correct and accurate, and you chased me around the bush back to the beginning, and asked me to change it basically almost word for word what I first Pull Requested...I am sorry, but this is frustrating.

Reply:
Your first commit is this, which is quite different from my suggestion where it describes root_node.
https://github.com/godotengine/godot/pull/93752/commits/fbfddff90eac6e6572bd9bbd12bde31673b2bc2a

Reply:
you mean this part "The parent node is [i]usually[/i] the [member root_node]." ?

Reply:
Oh I see my error....my apoligies